test/d_do_test.d

```diff
@@ -416,7 +418,8 @@ bool compareOutput(string output, string refoutput)
             return false;
         refoutput = refoutput[pos + 3 ..$];
         output = output[pos..$];
-        munch(output, "0123456789");
+        auto p = output.byCodeUnit.countUntil!(e => digits.canFind(e));
```

Comment:
In theory `countUntil("0", "1", "2", "3", "4", "5", "6", "7", "8", "9")` might be marginally faster, but it shouldn't matter at all here.